### ISSUE TITLE
Boxplot of the swim distances of all eels per month

### ISSUE THREAD

**Issue by stijnvanhoey** [PieterjanVerhelst/resident-eel-polder-analysis#2]
In the paper, the boxplot with swim distances in combination with the temperature plot has also the number of eels taken into account on top of the different boxplots. 

We should add this in the R-code itself (annotate function of ggplot) to overcome the manual manipulation of the figure.

Furthermore, the addition of the pairwise relation to the graph, any suggestion?

**Comment by stijnvanhoey** [comment]
Same for the 'Boxplots of swim speed (mh-1) in the four diurnal'

**Comment by PieterjanVerhelst** [comment]
Indeed. I am still figuring out how I can illustrate the pairwise relationships for the boxplot with swim speeds over different months (cfr. the letters in the boxplot with swim speeds in the four diurnal phases).

**Comment by stijnvanhoey** [comment]
Is the usage of A, B, C,... clear enough to the reader? What do they represent with respect to your data?

Concerning the visualisation of pariwise relationships, a heatmap seems most appropriate?
However, if you want to combine it with the boxplots, some ideas/proof of concept:
- add heatmap to the boxplot (maybe * for significance)
![image](https://cloud.githubusercontent.com/assets/754862/18671005/a7c1228e-7f42-11e6-8e15-2ac8a156f09d.png)
- or integrate the boxplots inside the heatmap (but you lose comparison in between month values)
![image](https://cloud.githubusercontent.com/assets/754862/18671205/737c748c-7f43-11e6-9464-97de73cad647.png)
- Maybe it looks too much like a dendrogram, but connecting the different items with lines, adding the value:
![image](https://cloud.githubusercontent.com/assets/754862/18671400/338f7e22-7f44-11e6-8062-e551a54f54fb.png)

Just some ideas to think about

**Comment by PieterjanVerhelst** [comment]
I found a paper where letters were used to indicate the pairwise relationships: De Meester et al. (2016) Cryptic diversity and ecosystem functioning: a complex tale of differential effects on decomposition. However, your first idea looks interesting, though a little bit bulky. Is this still manageable for the plot with monthly swim distances?
